server/api/blog.controller.js

```diff
@@ -79,10 +79,10 @@ exports.newBlog = async (req, res) => {
 };
 
 exports.getPageOfBlogPosts = (req, res) => {
-  if (isNaN(req.params.page) || req.params.page < 1) {
+  if (isNaN(req.query.page) || req.query.page < 1) {
```

Comment:
Here we should use query params for pagination instead or params. Example: `/blogs?page=1`

This may actually need to be changes to `req.query.params.page` not sure of the syntax